maxi-blocks/maxi-blocks
# Issue-#2459 Add ResponsiveTabs breakpoitns

Fixes : #2459 

**How to test:**
1- Add your block
2- open style collection(border, alignemnt ....) check if reponsiveTabControl is present in the top of this inspctor accordion.
3- switch to a breakpoint and change the style : check if the indicator is added to the reponsiveTabControl 👍
